src/common/log.ts

```diff
@@ -1,15 +1,45 @@
 import * as riverpig from 'riverpig'
+import { LoggerConfig } from 'riverpig'
+import * as debug from 'debug'
 
 import through2 = require('through2')
 const logStream = through2()
 logStream.pipe(process.stdout)
 
 // TODO: Not clear why I needed this, but got a
 //   TypeScript error without it.
-export interface ConnectorLogger extends riverpig.Logger { }
+export class ConnectorLogger {
```

Comment:
you could extend riverpig.Logger and add only the `trace` function

Reply:
The riverpig.Logger  that is exported is a type alias, rather than a class. It doesn't appear to export its own Logger class. I think extending this will still require me to define a class that fits the extended type definition